Ports/harfbuzz/package.sh

```diff
@@ -5,17 +5,22 @@ files="https://github.com/harfbuzz/harfbuzz/releases/download/${version}/harfbuz
 useconfigure='true'
 auth_type='sha256'
 depends=("freetype" "libicu")
-configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt" "-DHB_HAVE_FREETYPE=ON" "-DHB_HAVE_ICU=ON")
+configopts=(
+	"-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+	"-DHB_HAVE_FREETYPE=ON"
+	"-DHB_HAVE_ICU=ON"
+	"-DCMAKE_C_FLAGS=-lfreetype"
+	"-Bbuild"
+)
 
 configure() {
-    run mkdir -p build
-    run sh -c "cd build && cmake .. ${configopts[@]}"
+    run cmake ${configopts[@]}
 }
 
 build() {
-    run sh -c "cd build && make ${makeopts[@]}"
+    run sh -c "cd build; make ${makeopts[@]}"
 }
 
 install() {
-    run sh -c "cd build && make install"
+    run sh -c "cd build; make install"
```

Comment:
Why change `&&` to `;` here and in `build()`? The idiomatic way is to chain commands with `&&`.